developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md

```diff
@@ -11,11 +11,7 @@ The `aptos` tool is a command line interface (CLI) for developing on the Aptos b
 * [Installing the Move Prover](../install-move-prover.md) for an optional tool to validate your Move code.
 * [Using Aptos CLI](use-aptos-cli.md) for detailed instructions on employing the Aptos CLI.
 
-To download the Aptos source code, related tools, and IDE plugins for the Move programming language, follow the instructions below. Whenever possible, install the CLI by downloading the precompiled binary for your platform, as described below. If needed, you may build from source or use one the other [methods of installation](./index.md).
-
-:::tip Use setup script
-Aptos offers the [`dev_setup.sh`](https://github.com/aptos-labs/aptos-core/blob/main/scripts/dev_setup.sh) script for establishing your development environment. This script currently supports macOS and Ubuntu Linux with other Linux distributions working but untested. The script does not support Windows. See the instructions below to manually install necessary dependencies.
-:::
+Binary releases are recommended for most users, otherwise see [Building Aptos From Source](../../guides/building-from-source.md)
```

Comment:
Wondering whether the otherwise statement should point to the many other options for installing the CLI:
https://aptos.dev/cli-tools/aptos-cli-tool/

Reply:
Maybe [Binary releases] should link to that?

Reply:
Maybe the eventual split for binary installation will be "Install With Package Managers" (deb, rpm, brew, etc.) and "Install Binary Release Archive" and "Build From Source" ?

Reply:
Maybe we should push people towards installing from the script since it's one command and simple.  After that point they can just call `aptos update`, which we may also want to eventually mention somewhere for upgrading the CLI